.github/workflows/ci-ember-flight-icons.yml

```diff
@@ -31,12 +31,6 @@ jobs:
       - name: Lint
         run: yarn run lint
         working-directory: packages/ember-flight-icons
-      - name: Run Tests
-        run: yarn run test:ember:percy
-        working-directory: packages/ember-flight-icons
```

Comment:
We need to keep this section so the tests still run, but remove the `:percy` section from the yarn command

Reply:
or do we 🤔 ? They're also running as part of the ember-try scenarios so maybe this is correct and I was off base with my initial response.

Reply:
Right, we still have a few ember tests running

Reply:
yeah, I think the matrix test should cover this too

Reply:
I guess the only reason to keep it is that the version of ember specified in our project could be different than the `ember-release` scenario.

I think what you have here is good now but it would be worth discussing with the team whether it makes sense for us to run these tests 3 times.

Reply:
@hashicorp/hds-engineering

Reply:
could we imagine (would it be possible) to move these tests inside the `packages/components` test suite, like I did for the Percy test? Or because it's an external component (coming from an npm package, I mean) they would not work?

Reply:
I think technically it would work since one depends on the other already (it'd be easy to try and see). It's definitely _unorthodox_ to have tests for a package inside another package, but given how our packages are effectively tied together and we never change the ember-flight-icons package the risks seem minimal.

This opens up another can of works around whether or not it make sense for these to be separate packages at all, but we could consider that separately...

Reply:
> This opens up another can of works around whether or not it make sense for these to be separate packages at all, but we could consider that separately...

It doesn't make sense to make them separate, I don't think it's a can of worms, and we should make a story to combine them :)

Reply:
It sounds like we're in agreement that running the `ember-flight-icons` tests separately doesn't make much sense, so I [moved the icons tests to our components package](https://github.com/hashicorp/design-system/pull/1218/commits/48a3dec640c2e9710eaad2aa2d230345db65e9a9) and [removed the `ember-flight-icons` CI testing](https://github.com/hashicorp/design-system/pull/1218/commits/a477f989dcb2f0fb4893fd2b4a210afe36979025). 

I can also record a card to look into/investigate the impact of moving the `ember-flight-icons` into `components`.